src/GrammarParser.php

```diff
@@ -92,12 +112,12 @@ public function buildGrammar($grammarStr, $options = [])
                 $branchName = (string)$grammarBranch->findFirst('branchName');
                 $branchTypeNodeStr = (string)$grammarBranch->findFirst('branchType');
                 $branchType = $branchTypeNodeStr ? substr($branchTypeNodeStr, 1, -1) : $defaultBranchType;
-                $grammar[$branchName] = \ParserGenerator\GrammarNode\BranchFactory::createBranch($branchType,
-                    $branchName);
-            } else {
```

Comment:
Un-nesting

Reply:
I don't like it.

I think old fashion 
if (condition) {
  codetrue
} else {
  codefalse
}

is more readable than:

if (condition) {
  codetrue
  continue;
}
codefalse

Reply:
👍  => undid the change!